lib/fizzy/execute.cpp

```diff
@@ -685,142 +469,6 @@ std::optional<uint32_t> find_export(const Module& module, ExternalKind kind, std
 
```

Comment:
Where is the `find_export` helper above used? I always thought it is used by instantiation, but it is used in the execution loop?!

Reply:
No, it's used only in ``find_exported_*` API

Reply:
Those operate on the instance, shouldn't they belong into `instantiate.cpp`?

Reply:
Logically they are neither instantiation nor execution, I would put them into separate header, maybe `export.hpp`, or `api.hpp` because we have `api_test.cpp` for them. What do you think?

Reply:
It is hard for me to discriminate what they are for (not my area of knowledge) so "kinda whatever". Can be also "module.hpp/cpp" if that make sense (I know this is non-functional name, but this can be found in other projects).

But I really want to have only execute() function in execute.cpp what makes execution optimizations easier to conduct. In the end we can think about it as: extracting execute() from the instatiate.cpp file...

Reply:
That was getting a bit too complicated for one PR, will keep them in instantiate.